packages/nimble-components/src/table-column/base/models/column-validator.ts

```diff
@@ -0,0 +1,55 @@
+import type { ValidityObject } from '../../../table/types';
+import type { ColumnInternals } from './column-internals';
+
+type ObjectFromList<T extends readonly string[], V = string> = {
```

Comment:
Can we get rid of `V` and just write this to use `boolean` rather than `V`? It is only ever used within this file and only ever used as a `boolean`.